src/main/java/teammates/storage/sqlsearch/AccountRequestSearchDocument.java

```diff
@@ -0,0 +1,37 @@
+package teammates.storage.sqlsearch;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import teammates.storage.sqlentity.AccountRequest;
+
+/**
+ * The {@link SearchDocument} object that defines how we store document for
+ * account requests.
+ */
+class AccountRequestSearchDocument extends SearchDocument<AccountRequest> {
+
+    AccountRequestSearchDocument(AccountRequest accountRequest) {
+        super(accountRequest);
+    }
+
+    @Override
+    Map<String, Object> getSearchableFields() {
+        Map<String, Object> fields = new HashMap<>();
+        AccountRequest accountRequest = entity;
+        String email = accountRequest.getEmail();
+        String institute = accountRequest.getInstitute();
+
+        String[] searchableTexts = {
+                accountRequest.getName(), email, institute,
+        };
+
+        fields.put("id", email + '%' + institute);
```

Comment:
id should be the string representation of the new UUID of the object. same for the other entities.